src/components/utils/svgA11yHidden.ts

```diff
@@ -0,0 +1,6 @@
+export const svgA11yHidden = {
+    // Hides element from a11y tree
+    'aria-hidden': true,
+    // Prevents focus in IE
+    focusable: false,
```

Comment:
We don't support IE, so there is no need to add this.

Reply:
I guess it is not a big deal, we improve UX for IE users, they will receive redundant focus instead

Reply:
Redundant focus will be the least problem. JS simply won't work in IE due to modern features we are using in the code. No support for IE - no legacy code